src/pages/CarDetails/CarDetails.scss

```diff
@@ -1,32 +1,209 @@
+@import "./common-fontstyles.scss";
+@import "../../themes/colors.scss";
+
 .totaldiv {
-  margin: 2rem;
+  padding-top: 1rem;
+  border-left: 8vh solid black;
+  border-right: 8vh solid black;
```

Comment:
why do we need to use vh??

Reply:
to get the left and right vertical black border i have used 8vh 

![image](https://user-images.githubusercontent.com/96569364/166189832-9308711d-07c8-485f-b14f-2e1735672db6.png)